patches/api/0480-Allow-new-map-cursor-type-in-map-cursor.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Lulu13022002 <41980282+Lulu13022002@users.noreply.github.com>
+Date: Fri, 24 May 2024 20:19:07 +0200
+Subject: [PATCH] Allow new map cursor type in map cursor
+
+
+diff --git a/src/main/java/org/bukkit/map/MapCursor.java b/src/main/java/org/bukkit/map/MapCursor.java
+index c99311aa31ee293f4e61bca4bc3f401bb20f42f9..a046d32fb34fa1ede6f6141f0ca8e2f7b7ec1eb4 100644
+--- a/src/main/java/org/bukkit/map/MapCursor.java
++++ b/src/main/java/org/bukkit/map/MapCursor.java
+@@ -221,8 +221,9 @@ public final class MapCursor {
+      */
+     @Deprecated(forRemoval = true, since = "1.20.2") // Paper
+     public void setRawType(byte type) {
+-        if (type < 0 || type > 26) {
+-            throw new IllegalArgumentException("Type must be in the range 0-26");
++        class Bounds { public static final int UPPER_MAP_CURSOR_TYPE = Type.values().length - 1; } // Paper
```

Comment:
Move this to a proper private field on mapcursor or type, this isn't an interface